src/stem.ts

```diff
@@ -7,116 +7,156 @@
 import { Vex } from './vex';
 import { Element } from './element';
 import { Flow } from './tables';
+import { BoundingBox } from './boundingbox';
 
 // To enable logging for this class. Set `Vex.Flow.Stem.DEBUG` to `true`.
-function L(...args) {
+function L(
+  // eslint-disable-next-line
+  ...args: any[]) {
   if (Stem.DEBUG) Vex.L('Vex.Flow.Stem', args);
 }
 
+export interface StemStruct {
```

Comment:
Let's call all options parameters `*Options` so 'StemOptions' here. (`NoteStruct` is a legacy wart to clean up.)

Reply:
ok